test/integ/deadline_mcp/test_mcp_server_integration.py

```diff
@@ -0,0 +1,174 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""Integration test for the Deadline Cloud MCP Server."""
```

Comment:
These look good. It'd be really great to add another test (or adapt this one) to create a real MCP client that starts the MCP server and issues commands. It would help verify that the fast MCP process is wired up correctly.

Reply:
Also add one integ test for error handling